src/Bicep.Core/Parser/ParseDiagnosticsVisitor.cs

```diff
@@ -25,6 +25,12 @@ public override void VisitProgramSyntax(ProgramSyntax syntax)
             base.VisitProgramSyntax(syntax);
             
             this.diagnosticWriter.WriteMultiple(syntax.LexerDiagnostics);
+
+            var targetScopeSyntaxes = syntax.Children.OfType<TargetScopeSyntax>().ToList();
+            foreach (var targetScope in targetScopeSyntaxes.Skip(1))
```

Comment:
>Skip [](start = 60, length = 4)

The first one isn't necessarily the correct one. We should mark all of them as duplicate just like we do with identifiers. (Sorry I may have misled you into doing that when we talked.)